docs/src/api.md

```diff
@@ -303,3 +313,10 @@ InputOutput.read_field
 ```@docs
 Remapping.interpolate_array
 ```
+
+## Other
+
+
+```@docs
+Limiters.quasimonotone_limiter
```

Comment:
We already have the `Limiters` section with this API docstring 
https://clima.github.io/ClimaCore.jl/previews/PR1419/api/#ClimaCore.Limiters.QuasiMonotoneLimiter
Why add an "Other" section?

Reply:
And the latter one added errors:
![image](https://github.com/CliMA/ClimaCore.jl/assets/39932030/9e1f3a1d-b8d3-41e0-a037-0482256019c5)

Reply:
Yes, but `quasimonotone_limiter` is not in the API section. Actually, I misinterpreted the error--
```julia
"""
    AbstractLimiter

Supertype for all limiters.

# Interfaces

- [`quasimonotone_limiter!`](@ref)
"""
abstract type AbstractLimiter end
```

`quasimonotone_limiter!` is referencing a non-existing function. I think it's supposed to be `apply_limiter!`.